triples/package.json

```
@@ -48,7 +48,7 @@
   },
   "devDependencies": {
     "@napi-rs/cli": "workspace:*",
-    "@types/node": "^20.11.13",
+    "@types/node": "^22.7.4",
```

Can the dependencies be updated?

```suggestion
    "@types/node": "^22.17.0",
```